app.js

```diff
@@ -63,4 +63,268 @@ function fillData() {
   textArea.value = desirializedObject.textArea;
 }
 
-window.addEventListener('load', fillData);
\ No newline at end of file
+window.addEventListener('load', fillData);
+
+// create the card information
+const otherProjects = [
+  {
+    name: 'Multi-Post Stories',
+    description:
+      "A daily selection of privately personalized reads; no accounts or sign-ups required. has been the industry's standard dummy text ever since the 1500s, when an unknown printer took a standard dummy text.",
+    image: './assets/first-card.png',
+    desktopImg: './assets/large.png',
+    technologies: ['css', 'html', 'bootstrap', 'Ruby'],
+    liveLink: '#',
+    sourceLink: '#',
+  },
+  {
```

Comment:
- [ ] Great job. please centralize your mobile version image so it looks like the figma version
- [ ] kindly adjust the tech buttons as well. also, ensure the see source and see live buttons are of same size.

yours
![Screenshot (604)_LI](https://user-images.githubusercontent.com/65298048/202849239-e3d032ed-4eb2-4d58-835a-5ff8e47d207c.jpg)
figma
![Screenshot (600)](https://user-images.githubusercontent.com/65298048/202849270-54b283b2-a3e7-43af-bebb-8ef17a6aae4f.png)